src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -51,16 +51,11 @@ private void captureOs() {
 
     private void captureIde() {
         if (!isCi()) {
-            Optional<String> newIdeaVersion = sysProperty("idea.version");
-            Optional<String> oldIdeaVersion = firstSysPropertyKeyStartingWith("idea.version");
```

Comment:
Please optimize the imports. And remove any methods that are now unused in the Utils class.

Reply:
done